### DISCUSSION TITLE
Feder: visualization tool for faiss ivf flat index

### DISCUSSION THREAD

**Discussion by shanghaikid** [facebookresearch/faiss#2348] [Show and tell]
We are building something really cool to let people make sense of the search process of similarity search. and It's open source!

[Project feder](https://github.com/zilliztech/feder)

## examples
[Javascript example](https://observablehq.com/@min-tian/feder)
[Jupternotebook example](https://colab.research.google.com/drive/12L_oJPR-yFDlORpPondsqGNTPVsSsUwi#scrollTo=N3qqBAYxYcbt)

![image](https://user-images.githubusercontent.com/185051/172820367-c9920570-2bd0-43fd-bc57-87d36233dcc8.png)
![image](https://user-images.githubusercontent.com/185051/172820374-5c2156ea-07a1-4d6f-ad40-165904254ae6.png)

Any advice is welcome.

**Comment by qhyuan1992** [comment]
great!
